kratos/utilities/sparse_matrix_multiplication_utility.h

```diff
@@ -68,26 +65,26 @@ class SparseMatrixMultiplicationUtility
     KRATOS_CLASS_POINTER_DEFINITION( SparseMatrixMultiplicationUtility );
 
     /// The size type
-    typedef std::size_t SizeType;
+    using SizeType = std::size_t;
 
     /// The index type
-    typedef std::size_t IndexType;
+    using IndexType = std::size_t;
 
     /// The signed index type
-    typedef std::ptrdiff_t  SignedIndexType;
+    using SignedIndexType = std::ptrdiff_t;
 
     /// A vector of indexes
-    typedef DenseVector<IndexType> IndexVectorType;
+    using IndexVectorType = DenseVector<IndexType>;
 
     /// A vector of indexes (signed)
-    typedef DenseVector<SignedIndexType> SignedIndexVectorType;
+    using SignedIndexVectorType = DenseVector<SignedIndexType>;
 
     ///@}
     ///@name Life Cycle
     ///@{
 
     /// Default constructor
-    SparseMatrixMultiplicationUtility(){};
+    SparseMatrixMultiplicationUtility() = default;
```

Comment:
very minor nitpicking: do we ever actually instantiate this class? If not, why not make it unambiguous and explicitly delete all constructors and destructors?